lib/twine/cli.rb

```diff
@@ -11,7 +11,7 @@ def self.parse_args(args, options)
       new(args, options).parse_args
     end
 
-    def parse_args
+    def parse_args    # TODO: let this method take args and return options
```

Comment:
Should this TODO be left in here? I'm not a fan of TODOs in code unless they are critical and/or obvious.